_data/accessibility-tests/prose.yml

```diff
@@ -0,0 +1,37 @@
+title: Prose
+component_status: pass
+test_items:
+# General tests
+  - summary: Information remains usable after style changes.
+    summary_additional: After you adjusts text spacing, you can still access and use all of the information on the page. You can see that content adjusts correctly in response to styling changes.
+    test_status: pass
+    test_type: general
```

Comment:
I think 1.4.4 can indeed cover this; I'd hate to introduce questions about text spacing when we may not have answers just yet. I say let's remove that check for now.

Reply:
@amycole501  do you have a preference for removing it entirely or replacing it with the screen magnification item that Amy suggested above?

Reply:
Sorry I should have clarified; 1.4.4 can replace it.

Reply:
Yes that's the best route so that check is covered.

Reply:
@mahoneycm FYI one of our goals for the near future is to provide instructions for manipulating text styles in a new "how to" accordion. Until that is ready, we are temporarily marking the text style tests as zoom tests OR temporarily excluding them from the site (depending on its similarity to other tests on the page). 

@amycole501 and @alex-hull, correct me if I'm wrong, but to me that means that we should keep them in the checklist spreadsheet because we are still testing for these internally. The choice to remove 1.4.4 here is for the website _only_.

Reply:
Sorry @amycole501 I did the "comment without refreshing" thing again 🤦 . I don't want to muddy the waters here, so whatever @amycole501 says is correct!

Reply:
Ok removed the `1.4.12` criterion from the _site only_ in [6ce47f0](https://github.com/uswds/uswds-site/pull/2613/commits/6ce47f00c0e1478c7f2154f8e53e19356d253465).

My previous question can be ignored because I didn't realize that the `1.4.4` item was already present in both site checklist and the critical testing checklist